src/models/transaction.ts

```diff
@@ -92,6 +93,32 @@ class Transaction {
     this._dataToSignCache = null;
   }
 
+  /**
+   * Get the signature from the dataToSignHash for a private key
+   *
+   * @return {Buffer}
+   * @memberof Transaction
+   * @static
+   *
+   */
+  static sign(dataToSignHash: Buffer, privateKey: PrivateKey): Buffer {
+    return transactionUtils.sign(dataToSignHash, privateKey);
+  }
+
+  /**
+   * Get the signature of this transaction for the given private key
+   *
+   * @param {PrivateKey}
+   *
+   * @return {Buffer}
+   * @memberof Transaction
+   * @inner
+   *
+   */
+  sign(privateKey: PrivateKey): Buffer {
```

Comment:
I would change the method name, this is `getSignature`, you are not signing the transaction.

Reply:
Done